gum/backend-linux/gumprocess-linux.c

```diff
@@ -861,6 +865,9 @@ gum_emit_executable_module (const GumModuleDetails * details,
 {
   GumEmitExecutableModuleContext * ctx = user_data;
 
+  if (!gum_handle_interpreter (details, user_data))
```

Comment:
```suggestion
  if (!gum_maybe_emit_interpreter (details, user_data))
```

To make it clear that the function is related